tests/display.rs

```diff
@@ -350,5 +350,5 @@ fn debug_string() {
     use glium::backend::Facade;
 
     let display = support::build_display();
-    display.get_context().insert_debug_marker("Hello world");
+    display.get_context().insert_debug_marker("Hello world").unwrap();
```

Comment:
More than the API, the test also checks that the function doesn't trigger any OpenGL error.